src/ReleaseHistory.md

```diff
@@ -412,4 +412,7 @@
 ## **v2.1.6** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.6) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.6) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.6) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.6)
 * BUGFIX: Fortify FPR converter does not populate originalUriBaseIds if the source is a drive letter (e.g. C:)
 * BUGFIX: Multitool rebaseUri command throws null reference exception if results reference run.artifacts using the index property.
-* BUGFIX: Pre-release transformer does not upgrade schema uri if input version is higher than rtm.1.
\ No newline at end of file
+* BUGFIX: Pre-release transformer does not upgrade schema uri if input version is higher than rtm.1.
+
+## **v2.1.7** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.7) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.7) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.7) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.7)
+* BUGFIX: Multitool rewrite InsertOptionalData operations fail if results reference run.artifacts using the index property.
```

Comment:
>run.arti [](start = 84, length = 8)

Can we please retain the discipline of using markdown backticks to encapsulate reference to properties? Open this file in any markdown viewer to see how these new items compare to previous,

`run.artifacts` 

there are other instances of this NIT in this file.

Reply:
thanks! noted ! :) i will update

---
In reply to: [299195692](https://github.com/microsoft/sarif-sdk/pull/1551#discussion_r299195692) [](ancestors = 299195692)